enhancements/kube-apiserver/stability-history-of-resources.md

```diff
@@ -0,0 +1,201 @@
+---
+title: stability-history-of-resources
+authors:
+  - "@deads2k"
+reviewers:
+approvers:
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+see-also:
+  - "/enhancements/this-other-neat-thing.md"  
+replaces:
+  - "/enhancements/that-less-than-great-idea.md"
+superseded-by:
+  - "/enhancements/our-past-effort.md"
+---
+
+# Stability: Keep History of Resources
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions [optional]
+
+This is where to call out areas of the design that require closure before deciding
+to implement the design.  For instance, 
+ > 1. This requires exposing previously private resources which contain sensitive
+  information.  Can we do this? 
+
+## Summary
+
+When debugging a failed e2e test (or a string of them), one common question is, "what is the status of clusteroperator/foo
+when this particular test was running".
+While we could consider one-off solutions to this, we have a solution for storing this information inside of a local
```

Comment:
Could you link to the tools metioned here? This document as written is so vague as to not be understandable :|

Reply:
> Could you link to the tools metioned here? This document as written is so vague as to not be understandable :|

It captured the idea for @damemi who I think has found the tool and has a PR to add it.

Reply:
@stevekuznetsov the tool being referred to is https://github.com/mfojtik/ci-monitor-operator and we are working on adding it in https://github.com/openshift/origin/pull/24845

This has inspired a longer-term goal for me to add [distributed tracing[(https://github.com/openshift/enhancements/pull/279) throughout our components